### PR TITLE
add amendments from 2016 in-person meeting

### PR DESCRIPTION
**Pull request by JasonJWilliamsNY**
This will need review, and there are additional amendments forthcoming. These will be presented to the Advisory Counsel after the Steering Committee's ratification.

### CODE REVIEW

**Comment by karinlag** [comment]
Looks good to me.

**Comment by rgaiacs** [comment]
Looks good.

File: governance.md [hunk 1]
```diff
@@ -58,14 +58,12 @@ Governance
...
-    a.  The Advisory Council meets quarterly, either in person or online.
+    a.  The Advisory Council meets as determined by the chair, either in person or online.
 
     b.  Minutes of the Advisory Council's meetings must be published
         within one week of each meeting.
 
-    c.  Any organization that qualifies as a Partner or Affiliate
-        under the organizational membership rules may appoint a
-        representative to the Advisory Council.
+    c.  Any organization that qualifies as a Partner at the 'Silver' level may appoint a representative to the Advisory Council.
```
Comment (jduckles) [inline governance.md]: 'Silver' level or above

**Comment by raynamharris** [comment]
looks good

**Comment by k8hertweck** [comment]
Is this ready to be merged?

**Comment by rgaiacs** [comment]
I think that @JasonJWilliamsNY is waiting some feedback from the advisory board.

**Comment by rgaiacs** [comment]
@JasonJWilliamsNY Any update on this one?

**Comment by k8hertweck** [comment]
I'm assuming no news is good news and merging these mostly minor changes.

**Pull request merged**